spec/lib/sufia/user_stat_importer_spec.rb

```diff
@@ -46,61 +65,92 @@
-      OpenStruct.new(date: date_strs[3], pageviews: 14),
-      OpenStruct.new(date: date_strs[4], pageviews: 15)
+        OpenStruct.new(date: date_strs[0], pageviews: 11),
+        OpenStruct.new(date: date_strs[1], pageviews: 12),
+        OpenStruct.new(date: date_strs[2], pageviews: 13),
+        OpenStruct.new(date: date_strs[3], pageviews: 14),
+        OpenStruct.new(date: date_strs[4], pageviews: 15)
     ]
   }
 
   let(:frodo_file_1_pageview_stats) {
     [
-      OpenStruct.new(date: date_strs[0], pageviews: 2),
-      OpenStruct.new(date: date_strs[1], pageviews: 4),
-      OpenStruct.new(date: date_strs[2], pageviews: 1),
-      OpenStruct.new(date: date_strs[3], pageviews: 1),
-      OpenStruct.new(date: date_strs[4], pageviews: 9)
+        OpenStruct.new(date: date_strs[0], pageviews: 2),
+        OpenStruct.new(date: date_strs[1], pageviews: 4),
+        OpenStruct.new(date: date_strs[2], pageviews: 1),
+        OpenStruct.new(date: date_strs[3], pageviews: 1),
+        OpenStruct.new(date: date_strs[4], pageviews: 9)
+    ]
+  }
+
+  #work
+  let(:bilbo_work_1_pageview_stats) {
+    [
+        OpenStruct.new(date: date_strs[0], pageviews: 1),
+        OpenStruct.new(date: date_strs[1], pageviews: 2),
+        OpenStruct.new(date: date_strs[2], pageviews: 3),
+        OpenStruct.new(date: date_strs[3], pageviews: 4),
+        OpenStruct.new(date: date_strs[4], pageviews: 5)
+    ]
+  }
+
+  let(:bilbo_work_2_pageview_stats) {
+    [
+        OpenStruct.new(date: date_strs[0], pageviews: 11),
+        OpenStruct.new(date: date_strs[1], pageviews: 12),
+        OpenStruct.new(date: date_strs[2], pageviews: 13),
+        OpenStruct.new(date: date_strs[3], pageviews: 14),
+        OpenStruct.new(date: date_strs[4], pageviews: 15)
+    ]
+  }
+
+  let(:frodo_work_1_pageview_stats) {
+    [
+        OpenStruct.new(date: date_strs[0], pageviews: 2),
+        OpenStruct.new(date: date_strs[1], pageviews: 4),
+        OpenStruct.new(date: date_strs[2], pageviews: 1),
+        OpenStruct.new(date: date_strs[3], pageviews: 1),
+        OpenStruct.new(date: date_strs[4], pageviews: 9)
     ]
   }
 
   let(:bilbo_file_1_download_stats) {
     [
-      OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[0], totalEvents: "2"),
-      OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[1], totalEvents: "3"),
-      OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[2], totalEvents: "5"),
-      OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[3], totalEvents: "3"),
-      OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[4], totalEvents: "7")
+        OpenStruct.new(eventCategory: "Files", eventAction: "Downloaded", eventLabel: "bilbo1", date: date_strs[0], totalEvents: "2"),
```

Comment:
Sorry. Ran into problems then reverted but forgot to fix indents again. Will fix before next PR.